manifest.py

```diff
@@ -18,9 +18,6 @@
 
 source = os.path.join('src', 'cclib')
 files.append(os.path.join(source, "__init__.py"))
-files.append(os.path.join("src", "scripts", "ccget"))
```

Comment:
Any chance you could put these other fixes in a separate PR? Just keep the ones to fix ccwrite here.

Reply:
It's related to testing of ccwrite (is there a simpler way I'm missing?). I can move it to a different PR, but then this one will be left without a test.